srcCs/PerftestTransport.cs

```diff
@@ -119,6 +119,20 @@ public class PerftestTransport
         private const string TRANSPORT_CERTIFICATE_FILE_SUB = "./resource/secure/sub.pem";
         private const string TRANSPORT_CERTAUTHORITY_FILE = "./resource/secure/cacert.pem";
 
+        private string _LatencyTopicName;
+        private string _AnnouncementTopicName;
+        private string _ThroughputTopicName;
+        
+        
+        public bool useMulticast;
+        
+        public string latencyMulticastAddr;
+        public string announcementMulticastAddr;
+        public string throughputMulticastAddr;
+        
+        public SortedDictionary<string, string> topicNameMap = 
+            new SortedDictionary<string, string>();
```

Comment:
Can you add 4 spaces more before new, it should be 8 spaces. Result:
public SortedDictionary<string, string> topicNameMap = 
        new SortedDictionary<string, string>();